src/amo/components/Header/index.js

```diff
@@ -93,6 +93,14 @@ export class HeaderBase extends React.Component {
             >
               <DropdownMenuItem>{i18n.gettext('My Account')}</DropdownMenuItem>
               <DropdownMenuItem>
+                <Link
+                  className="Header-user-menu-collections-link"
+                  href={`/collections/${username}/`}
+                >
+                  {i18n.gettext('My Collections')}
```

Comment:
Can we change the text? We did `Verb Noun` for the other links in the section.

Reply:
Good point; "View Collections" is vague, but "View My Collections" could work.

Reply:
I wanted to suggest "View My Collections" too.